generator/booking_quality_check/booking_quality_check.cpp

```diff
@@ -73,15 +74,19 @@ int main(int argc, char * argv[])
     for (size_t const j : bookingIndexes)
     {
       auto const & hotel = bookingDataset.GetHotel(j);
-      double const distanceMeters = ms::DistanceOnEarth(e.lat, e.lon, hotel.lat, hotel.lon);
-      double score = BookingDataset::ScoreByLinearNormDistance(distanceMeters);
+      auto const score = booking_scoring::Match(hotel, e);
 
-      bool matched = score > BookingDataset::kOptimalThreshold;
+      double const distanceMeters = ms::DistanceOnEarth(e.lat, e.lon, hotel.lat, hotel.lon);
+      bool matched = score.IsMatched();
 
       outStream << "# ------------------------------------------" << fixed << setprecision(6)
                 << endl;
       outStream << (matched ? 'y' : 'n') << " \t" << i << "\t " << j
-                << " distance: " << distanceMeters << " score: " << score << endl;
+                << "\tdistance: " << distanceMeters
```

Comment:
What is the point to write tags like "distance", "distance score", etc again and again, instead of writing them once, as the header?